### PR TITLE
Task/refactor

### PR DESCRIPTION
**Pull request by horaklukas**
I've transferred package `ackee-image-server-sdk` to this repo. The idea is that it will be multi package repo, managed with [Lerna](https://github.com/lerna/lerna). 

This  means that source packages will be available in `packages` folder, so the structure will be
```
packages/
   resizin/
   node-resizin/
   react-resizin/
```

For now I consider every package name will be under @ackee scope, eg. @ackee/node-resizin, what do you think? Until now all image server related packages was "prefixed" with `ackee-` so there will be no major change in name, except that it finally unify the name with other platforms sdk.

For this PR I didn't do any code refactoring, only renamed files from `.js` to `.ts`, setup build and covered the library with tests. I also added some TODOs about potential improvements and maybe issues/bugs.

Code refactor and documentations improvements will follow in next PRs as well as `node-resizin` package 🙂 feel free to ask and oppose any changes.

### CODE REVIEW

File: packages/resizin/src/upload.ts [hunk 1]
```diff
@@ -0,0 +1,48 @@
+import fetch from 'isomorphic-fetch';
+import NodeFormData from 'form-data';
+import Promise from 'promise-polyfill';
+
+export const uploadImage = (url?: string, api_key?: string, id?: string, file?: string) => {
```
Comment (baierjak) [inline packages/resizin/src/upload.ts]: hezka funkcicka haha :))

File: tslint.json [hunk 2]
```diff
@@ -0,0 +1,55 @@
+{
```
Comment (baierjak) [inline tslint.json]: typescript nezkontroluju protoze ho skoro vubec neznam bohuzel

Comment (horaklukas) [reply tslint.json]: tohle je jen lint config

File: packages/resizin/src/upload.ts (hunk 1 above)
Comment (horaklukas) [reply packages/resizin/src/upload.ts]: ty volitelný parametry tady jsou vlastně jen kvuli testům, ale pak jsem přišel na to jak to udělat i bez toho, upravim 👍

File: scripts/build.sh [hunk 3]
```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
```
[the remark this answers is not in the document]
Comment (horaklukas) [reply scripts/build.sh]: Nepotřebujem, momentálně nemám v plánu dělat dist bundle a jakýkoliv loadery jsou zbytečný..

**Pull request merged**